CrazyCanvas/Source/ECS/Systems/Player/WeaponSystem.cpp

```diff
@@ -133,7 +137,7 @@ void WeaponSystem::Fire(WeaponComponent& weaponComponent, const glm::vec3& start
 		/* Scale */				pECS->AddComponent<ScaleComponent>(projectileEntity, {true, { 0.3f, 0.3f, 0.3f }}),
 		/* Rotation */			pECS->AddComponent<RotationComponent>(projectileEntity, {true, direction}),
 		/* Mesh */				pECS->AddComponent<MeshComponent>(projectileEntity, {m_ProjectileMeshComponent}),
-		/* Shape Type */		EShapeType::TRIGGER,
+		/* Shape Type */		EShapeType::SIMULATION,
 		/* CollisionGroup */	FCollisionGroup::COLLISION_GROUP_DYNAMIC,
 		/* CollisionMask */		FCollisionGroup::COLLISION_GROUP_PLAYER | FCollisionGroup::COLLISION_GROUP_STATIC,
```

Comment:
Could you change this to FCollisionGroup::COLLISION_GROUP_OTHER, to prevent the projectile from self collide with the user.

Reply:
You already did this in your branch and this branch doesn't change it. Also, since the projectiles' staring positions are offset from the player entity, projectiles won't collide with the players that fired them (initially).

Reply:
I meant to change the CollisionGroup to FCollisionGroup::COLLISION_GROUP_OTHER instead of FCollisionGroup::COLLISION_GROUP_PLAYER